views/angular_partials/car_select.slim

```diff
@@ -1,31 +1,37 @@
-.text-center
-  h1 I want to save money on my...
-  br
+.text-center.container
```

Comment:
Inside `layout.slim` there is a `.container-fluid` so this `.container` is overwriting that. Only need one so can get rid of this one.